docs/fast_data/connectors/debezium_cdc.md

```diff
@@ -0,0 +1,420 @@
+| DATABASE_HOSTNAME         | true     | Hostname or IP address of the database Debezium should connect and monitor changes                                           | -             |  
+| DATABASE_PORT             | true     | Database port on which Debezium should connect to                                                                            | -             |  
+| DEBEZIUM_DB_USERNAME      | true     | Debezium account username for accessing the database                                                                         | -             |  
+| DEBEZIUM_DB_PASSWORD      | true     | Debezium account password for accessing the database                                                                         | -             |  
+| TOPIC_PREFIX              | true     | Prefix to be added to each table on the sink topics where database changes are published                                     | -             |  
+| DATABASES_LIST            | true     | Comma separated list of database names or regular expression that defines which databases should be monitored                | -             |  
+| TABLES_LIST               | true     | Comma separated list of tables names (with their schema) or regular expression that defines which tables should be monitored | -             |  
+| KAFKA_BROKERS             | true     | Comma separated list of nodes address belonging to a Kafka cluster                                                           | -             |
+| KAFKA_USERNAME            | true     | The Kafka username                                                                                                           | -             |
+| KAFKA_PASSWORD            | true     | The Kafka password                                                                                                           | -             |
+| KAFKA_CLIENT_ID           | false    | Client identifier employed by this application                                                                               | -             |
+
+In the following section are provided a configuration example for each supported database and some instructions on how to configure and enable the database itself for CDC operations. Furthermore, it is clarified why Debezium Server CDC currently requires to be connected also to an instance of Redis.
+
+## Debezium Server Configuration
+
+In these paragraphs are detailed the configurations that have tested out with the Debezium Server plugin.  
+Currently, the default sink configuration is prefilled with Kafka properties, since it is the main event streaming platform supported by Fast-Data. Nonetheless, it is possible to fill it with any of the Debezium supported sink configuration as explained in the [official documentation](https://debezium.io/documentation/reference/2.1/operations/debezium-server.html#_sink_configuration). 
```

Comment:
```suggestion
## Debezium Server Configuration

In this section, some configuration examples for each supported database and some instructions on how to configure and enable the database itself for CDC operations are provided. Furthermore, it is clarified why Debezium Server CDC currently requires to be connected also to an instance of Redis.

In the next paragraphs, the configurations that have been tested out with the Debezium Server plugin are detailed.  
Currently, the default sink configuration is prefilled with Kafka properties, since it is the main event streaming platform supported by Fast-Data. Nonetheless, it is possible to fill it with any of the Debezium-supported sink configurations as explained in the [official documentation](https://debezium.io/documentation/reference/2.1/operations/debezium-server.html#_sink_configuration). 
```